sklearn/linear_model/ridge.py

```diff
@@ -867,10 +871,12 @@ def fit(self, X, y, sample_weight=None):
             #fit_params = {'sample_weight' : sample_weight}
             fit_params = {}
             gs = GridSearchCV(Ridge(fit_intercept=self.fit_intercept),
-                              parameters, fit_params=fit_params, cv=self.cv)
+                              parameters, fit_params=fit_params, cv=self.cv,
+                              scoring = self.scoring)
```

Comment:
please add a regression test for this.
That is, please compare the output of RidgeCV with cv != None to GridSearchCV, and make sure that the test fails on master.

Also, there shouldn't be a space before and after ``=``.